packages/typespec-go/test/azmodelsonly/zz_models.go

```diff
@@ -24,8 +19,3 @@ type DerivedTwo struct {
 	// REQUIRED
 	Name *string
 }
-
-type PreservedBaseClass struct {
```

Comment:
We need a fix in tcgc to resolve this. Tracked by https://github.com/Azure/typespec-azure/issues/1827